doc/conf.py

```diff
@@ -239,4 +239,28 @@
                }
 }
 
+suppress_warnings = [
+    'nbsphinx',
```

Comment:
What warnings are being suppressed here?

Reply:
Without this rendered notebooks will show info `DISPLAY` during initialization.